ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Footer.kt

```diff
@@ -48,21 +48,27 @@ internal fun Footer(
     templateConfiguration: TemplateConfiguration,
     viewModel: PaywallViewModel,
     childModifier: Modifier = Modifier,
+    allPlansTapped: () -> Unit = {},
 ) {
     Footer(
+        mode = templateConfiguration.mode,
         configuration = templateConfiguration.configuration,
         colors = templateConfiguration.getCurrentColors(),
         viewModel = viewModel,
         childModifier = childModifier,
+        allPlansTapped = allPlansTapped,
     )
 }
 
+@Suppress("LongParameterList")
 @Composable
 private fun Footer(
+    mode: PaywallMode,
     configuration: PaywallData.Configuration,
     colors: TemplateConfiguration.Colors,
     viewModel: PaywallViewModel,
     childModifier: Modifier = Modifier,
+    allPlansTapped: () -> Unit = {},
```

Comment:
I would remove this parameter, otherwise we might accidentally use it in a template and make the button do nothing.

Remember that users will be able to display all templates in all modes. I think we still need this to be optional and not display the button in templates where we don't support changing modes, like you had before?

Reply:
Yeah, I did that in the followup PR #1366 :)